src/fldeff_misc.c

```diff
@@ -800,16 +801,16 @@ static void Task_SecretBasePCTurnOn(u8 taskId)
     {
     case 4:
     case 12:
-        MapGridSetMetatileIdAt(data[0], data[1], 548);
+        MapGridSetMetatileIdAt(data[0], data[1], 0x224);
```

Comment:
Looks like you forgot about the ones in this file?

Reply:
The secret base ones I didn't name because I wasn't exactly sure how to handle it.
I though about naming them as being a part of the `gSecretBase` primary tileset, I don't remember why I didn't.